File: src/test/java/responsemodels/QueryResponseMessageTest.java
package responsemodels;

import static org.junit.Assert.assertEquals;

import org.junit.Before;
import org.junit.Test;

public class QueryResponseMessageTest {
  private static final String QUERY_MSG = "This is sample query msg.";
  private QueryResponseMessage queryResponseMessage;

  @Before
  public void setUp() {
    queryResponseMessage =
        new QueryResponseMessage.QueryResponseMessageBuilder()
            .setResponseMessage(QUERY_MSG)
            .build();
  }

  @Test
  public void getResponseMsg() {
    assertEquals(QUERY_MSG, queryResponseMessage.getData().getResponseMsg());
  }
}

File: src/test/java/responsemodels/QueryResponseMessageTest.kt
package responsemodels

import org.junit.Assert
import org.junit.Before
import org.junit.Test
import responsemodels.QueryResponseMessage.QueryResponseMessageBuilder

class QueryResponseMessageTest {
    private val QUERY_MSG = "This is sample query msg."
    private lateinit var queryResponseMessage: QueryResponseMessage

    @Before
    fun setUp() {
        queryResponseMessage = QueryResponseMessageBuilder()
            .setResponseMessage(QUERY_MSG)
            .build()
    }

    @Test
    fun responseMsg() {
        Assert.assertEquals(QUERY_MSG, queryResponseMessage.data!!.responseMsg)
    }
}